packages/engine/tests/networking/systems/OutgoingNetworkSystem.test.ts

```diff
@@ -255,6 +256,7 @@ describe('OutgoingNetworkSystem Unit Tests', () => {
         // make first entity the client's entity
```

Comment:
we should rename this test and buff it before merging to cover these changes

`should only queue client-owned state changes on client`
- assert that more than one client-owned object's state is queued